### DISCUSSION TITLE
Completing tasks on other .txt files (Question)

### DISCUSSION THREAD

**Discussion by NormCopeland** [todotxt/todo.txt-cli#382] [General]
I created a new file in my todo folder called ``work.txt``.  I see from the documentation how to add items to that list, as well as how to list them. But I can't figure out how to complete (``do|done``) items on extra lists.  I have the same question in regards to other actions such as ``app``, ``del`` etc.

Apologies if this has been written about elsewhere, I've searched and couldn't find anything. 

Thanks.

**Comment by inkarkat** [comment]
Task handling is centered around a single todo file; therefore, the support for a different todo file is minimal (just adding and moving). I guess the main use case was something like when you know that your todo file is outdated (no network, Dropbox down, etc.), but you still wanted to add a task. You could then `todo.sh addto` and later (after successfully catching up to the latest todo file) `todo.sh move` that task into the main `todo.txt`.

If you plan to use multiple todo files, please be aware that this is **discouraged**. However, it is possible to achieve this, via separate configuration files that override the `TODO_DIR` (this segregates _all_ todo files, also for done tasks and reports) or `TODO_FILE` (this just uses a separate file for the active tasks, but keeps all done tasks and reports in a single file) config variables. (You can `source` one config from the other, or separate common stuff into a third sub-config to avoid duplication; these are just Bash scripts like the `todo.sh` script itself.) You can create different aliases to easily switch between configurations (e.g. `alias todowork='todo.sh -d ~/.todo/work-config'`).
